packages/core/src/interpreter.ts

```diff
@@ -345,8 +345,12 @@ export class Interpreter<
         listener(errorEvent.data.data);
       });
     } else {
-      this.stop();
-      throw errorEvent.data.data;
+      if (this.parent) {
+        this.parent?.send(errorEvent);
```

Comment:
is it guaranteed that this event will always have the `origin` set? it feels important for clearing up the child in the parent

Reply:
this part is not needed anymore, it was introduced when developing so we should be good now